README.md

```diff
@@ -9,20 +13,44 @@ A modern approach to Neovim plugin management.
 
 ## :star2: Features
 
-- `Cargo`-like `rocks.toml` file for declaring all your plugins
-- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead)
-- Automatic dependency and build script management
+- `Cargo`-like `rocks.toml` file for declaring all your plugins.
+- Name-based installation (`use "nvim-neorg/neorg"` becomes `:Rocks install neorg` instead).
```

Comment:
suggestion: since packer is practically dead at this point

```suggestion
- Name-based installation (` "nvim-neorg/neorg" ` becomes `:Rocks install neorg` instead).
```

Reply:
Done!